### ISSUE TITLE
[airq] Air-Q reported density values scaled incorrectly

### ISSUE THREAD

**Issue by m0rgano** [openhab/openhab-addons#17809]
I haven't checked all the channels, but so far it appears that the density values being returned from the device are scaled incorrectly.

For example, the basic PM10 particulate density. This is from the 'fineDustConc10' channel.

This is of type Number:Density, which the base unit is kg/m³. 

A typical value for PM10 would be in µg/m³, however the reading being returned from the device is not scaling the reading, so for example a measurement on the device of 23µg/m³ is being received in openHAB as 23kg/m³.

To fix this, the incoming data values need to be scaled by a factor of 10 E-9.

This is on OH4.2.2.

**Comment by fwolter** [comment]
You're right, this seems to affect all density values. I'll take a look into it.
